rules/aws_cloudtrail_rules/aws_modify_cloud_compute_infrastructure.yml

```
@@ -0,0 +1,139 @@
+AnalysisType: rule
+Description: Detection when the cloud compute infrastructure is modified.
+DisplayName: AWS Modify Cloud Compute Infrastructure
+Enabled: true
+Filename: aws_modify_cloud_compute_infrastructure.py
+Reference: https://attack.mitre.org/techniques/T1578/
+Severity: High
+Tags:
+    - 'MITRE ATT&CK: T1578'
```

You'll want this to match [this format](https://github.com/panther-labs/panther-analysis/blob/cd220c87982011d4ad156c7daecd2857c358d154/rules/osquery_rules/osquery_linux_logins_non_office.yml#L13-L15) for this to show in the MITRE Matrix

```suggestion
Reports:
  MITRE ATT&CK:
    - TA0005:T1578
```